pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -2334,6 +2359,99 @@ def _impedance_calcul(self):
         )
         return self._impedance_l_w, self._impedance_w_l
 
+    def create_probe_port(self, reference_layer, rel_x_offset=0, rel_y_offset=0, r=0.01, name="Probe"):
+        """Create a coaxial probe port for the patch.
+
+        Parameters
+        ----------
+        reference_layer : class:`pyaedt.modeler.stackup_3d.Layer3D`
+            Reference layer (Ground).
+
+        rel_x_offset : float, value between 0 and 1
```

Comment:
Putting the value information here does not render properly in the doc. It should be part of the parameter description. The default should appear before the options in this format: The default is .... Lastly, using a list for the options, with a blank line before the first item and after the last, great improve the readability in the doc. I'd recommend fixing all of these descriptions.